packages/safe-tools-client/app/controllers/schedule.ts

```diff
@@ -0,0 +1,21 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class Schedule extends Controller {
+  @tracked isSetupSafeModalOpen = false;
+
+  @action openSetupSafeModal() {
+    this.isSetupSafeModalOpen = true;
+  }
+
+  @action closeSetupSafeModal() {
+    this.isSetupSafeModalOpen = false;
+  }
```

Comment:
I saw this set example and I found it very confusing to read, because the value is created on the controller, but then the template is the one "modifying", it is more concise, but it has mixed logic. If it's some type of pattern and ember required behavior I can switch to it.

Reply:
It’s a pattern but definitely not mandatory 🤓